lms/static/js/demographics_collection/DemographicsCollectionModal.jsx

```diff
@@ -200,27 +201,43 @@ class DemographicsCollectionModal extends React.Component {
           <Wizard.Header>
             {({ currentPage, totalPages }) => (
               <div>
-                <h2>Help make edX better for everyone!</h2>
-                <p>Thanks for registering with edX! Before getting started, please complete the additional information below to help your fellow learners. Your information will never be sold.</p>
+                <h2>
+                  {gettext('Help make edX better for everyone!')}
+                </h2>
+                <p>
+                  {gettext('Thanks for registering with edX! Before getting started, please complete the additional information below to help your fellow learners. Your information will never be sold.')}
+                </p>
                 <br />
                 <span className="fa-info-circle" />
-                <a className="pl-3">Why does edX collect this information?</a>
+                <a className="pl-3">
+                  {gettext('Why does edX collect this information?')}
+                </a>
                 <br />
-                <p>Part {currentPage} of {totalPages}</p>
+                <p>
+                  {StringUtils.interpolate(
+                      gettext('Part {currentPage} of {totalPages}'),
+                        { 
+                          currentPage: currentPage,
+                          totalPages: totalPages
+                        }
+                    )
+                  }
+                </p>
               </div>
             )}
           </Wizard.Header>
           <Wizard.Page>
             {({ wizardConsumer }) =>
               <div className="demographics-form-container">
+                {/* Gender Identity */}
                 <SelectWithInput
                   selectName={FIELD_NAMES.GENDER}
                   selectId={FIELD_NAMES.GENDER}
                   selectValue={wizardConsumer[FIELD_NAMES.GENDER]}
                   selectOnChange={this.handleSelectChange}
-                  labelText={"Gender identity"}
+                  labelText={gettext("What is your gender identity?")}
```

Comment:
Changed labels on the first page according to the Demographics questions doc: https://docs.google.com/document/d/1W4crGBigtgL_BUVoKzX6vmzz5p1mqR5ugWKTEusvWf4/edit

I confirmed with Ryan that the Modal should contain the full questions, which is different than the original mocks in Zeplin.

Reply:
Thanks for checking on this.